Integrations/integration-HashiCorp-Vault.yml

```diff
@@ -0,0 +1,1042 @@
+      description: Authentication policies
+    - contextPath: HashiCorp.Auth.LeaseDuration
+      description: 'Authentication lease duration in seconds, 0 if indefinitely '
+      type: number
+    description: Create a new token.
+  - name: hashicorp-get-policy
+    arguments:
+    - name: name
+      required: true
+      description: Policy name
+    outputs:
+    - contextPath: HashiCorp.Policy.Name
+      description: Policy name
+      type: string
+    - contextPath: 'HashiCorp.Policy.Rule.Path '
+      description: Policy rule path
+      type: string
+    - contextPath: HashiCorp.Policy.Rule.Capabilities
+      description: Policy rule capabilities
+    description: Get a policy with a specified name
+  - name: hashicorp-enable-engine
+    arguments:
+    - name: path
+      required: true
+      description: The path where the secrets engine will be mounted.
+    - name: type
+      required: true
+      description: Type of the backend, e.g. "aws".
+    - name: description
+      description: Human-friendly description of the mount.
+    - name: default_lease_ttl
+      description: The default lease duration, specified as a string duration like
+                     "5s" or "30m".
+    - name: max_lease_ttl
+      description: The maximum lease duration, specified as a string duration like
+                     "5s" or "30m".
+    - name: force_no_cache
+      description: Disable caching.
+    - name: audit_non_hmac_request_keys
+      description: Comma-separated list of keys that will not be HMAC'd by audit devices
+        in the request data object.
+      isArray: true
+    - name: audit_non_hmac_response_keys
+      description: Comma-separated list of keys that will not be HMAC'd by audit devices
+        in the response data object.
+      isArray: true
+    - name: listing_visibility
+      auto: PREDEFINED
+      predefined:
+      - unauth
+      - hidden
+      description: Whether to show this mount in the UI-specific listing endpoint.
+        Default is hidden.
+    - name: passthrough_request_headers
+      description: Comma-separated list of headers to whitelist and pass from the
+        request to the backend.
+      isArray: true
+    - name: kv_version
+      description: The version of the KV to mount. Set to "2" for mount KV v2.
+    - name: local
+      description: Specifies if the secrets engine is a local mount only. Local mounts
+        are not replicated nor (if a secondary) removed by replication. Supported
+        only in Vault Enterprise.
+    - name: seal_wrap
+      description: Enable seal wrapping for the mount. Supported only in Vault Enterprise.
+    description: Enable a new secrets engine at the given path.
+  - name: hashicorp-configure-engine
+    arguments:
+    - name: path
+      required: true
+      description: The engine path, e.g. secret/
+    - name: type
+      required: true
+      description: The engine type, e.g. KV
+    - name: version
```

Comment:
Shouldn't this be required or have an initial value?

Reply:
No, because it only applies to KV engine